### ISSUE TITLE
No indication if a menu is collapsed or expanded for Main Menus

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13561]
**Describe the bug**
- There is no indication (vocalization) if a menu is collapsed/expanded and which keys are needed for interaction.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS)
2. Tab to Navigation
3. Open menu using keyboard only
4. Hear error.

**Expected behavior**
- Using `aria-haspopup` and `aria-expanded` screen reader users should be able to hear if a menu is collapsed or expanded.
- Coding the expanding down-arrow as a button will convey to screen readers which keys the user has to use to expand and collapse de menu.
- There is already an Accessibility Pattern designed in Figma for the Main Menus, when ready to fix this, please ping me or @Manurod2018 for further details.

**Screenshots**

Currently:

![missing aria expanded](https://user-images.githubusercontent.com/15113219/131054535-7aa1570a-7d97-4c2f-ac16-68b4fae7c21d.jpg)

Example of expectation:

![menu collapsed](https://user-images.githubusercontent.com/15113219/131054547-bab1735c-4d4e-46da-9ea7-d274ed15e07f.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![menuCollapsed](https://user-images.githubusercontent.com/45409515/151868377-450d056c-4982-494f-aa7c-6bb687d6edae.png)
![menuExpanded](https://user-images.githubusercontent.com/45409515/151868382-5a28bfa7-a483-4bc0-add6-1d5e9a1f0bdc.png)
